src/main/webapp/app/pages/newsPage/UpdatedTxImplListItem.tsx

```diff
@@ -19,6 +23,26 @@ export const UpdatedTxImplListItem = (props: {
   )}`;
 
   const geneColumnIndex = 1;
+  const mutationColumnIndex = 2;
+
+  if (mutationColumnIndex > -1 && geneColumnIndex > -1) {
+    // transform the gene and mutation input to a link, ignore the inputs with comma, pipe or slash
+    props.data.forEach(row => {
+      const geneInput = row.content[geneColumnIndex].content;
+      const mutationInput = row.content[mutationColumnIndex].content;
+      if (typeof geneInput === 'string' && typeof mutationInput === 'string') {
+        if (!hasExcludedChars(mutationInput) && !hasExcludedChars(geneInput)) {
+          row.content[
+            mutationColumnIndex
+          ].content = convertGeneAndAlterationInputToLink(
```

Comment:
Why not directly use AlterationPageLink?

Reply:
for consistency and readability with the other func to convert gene input, but I agree its not worth to have a separate func for it